packages/@sparrow-workspaces/src/features/rest-explorer/components/response-body/ResponseBody.svelte

```diff
@@ -32,12 +33,17 @@
       : ''}"
   >
     {#if apiState.bodyLanguage === RequestDataTypeEnum.IMAGE}
-      <!-- 
-        --
-        -- Reponse content-type set to image,
-        -- 
-      -->
-      <SparrowLogo />
+      {#if !imageHasError}
+        <!-- render image from base64 -->
+        <img
+          src={`data:image/jpeg;base64,${response.body}`}
+          alt="Sparrow logo"
+          on:error={() => (imageHasError = true)}
+        />
+      {:else}
+        <!-- if any error render sparrow logo -->
+        <SparrowLogo />
```

Comment:
you can write a  message "Unable to load image." below sparrow icon here.

Reply:
Sure